features/OCP_WebConsole_insights.feature

```diff
@@ -0,0 +1,216 @@
+Feature: Insights displayed on OCP WebConsole
+
+
+  Scenario: Insights on OCP WebConsole for a cluster without any issue
+    Given console.redhat.com is accessible
+      And user U1 is already logged in into console.redhat.com
+      And user U1 switch to "OpenShift" page in console.redhat.com (OCM UI)
+      And user U1 is part of account (organization) with at least one cluster C1
+      And the cluster C1 sent insights-operator archive to the external data pipeline
+      And no issues have been detected for cluster C1
+     When user U1 selects "Clusters" item from the menu
+     Then new page with list of clusters should be displayed
+     When user U1 click on cluster name
+     Then a page with detailed information about the selected cluster should be displayed
+      And button named "Open Console" should be displayed
+     When user U1 click on "Open Console" button
+     Then new page named "Overview" with OCP WebConsole content should be displayed
+      And clickable "Insights" label should be displayed on that page
+      And information about no issues found should be displayed below "Insights" label
+     When user U1 click on "Insights" label
+     Then pop-up window named "Insights status" should appear
```

Comment:
It should be `"Insights Advisor status"` if I'm not wrong.

Reply:
nice catch, lemme fix it :)